bazel/cc_proto_descriptor_library/repro/repro_test.cc

```diff
@@ -0,0 +1,11 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+// Author: kmensah@google.com (Kwasi Mensah)
```

Comment:
not sure if we want to include this here?

Reply:
Whoops. Removed.